src/encoded/batch_download.py

```diff
@@ -72,7 +72,7 @@ def includeme(config):
     ('Library fragmentation method', ['files.replicate.library.fragmentation_method']),
     ('Library size range', ['files.replicate.library.size_range']),
     ('Biological replicate(s)', ['files.biological_replicates']),
-    ('Technical replicate', ['files.replicate.technical_replicate_number']),
+    ('Technical replicate', ['files.technical_replicates']),
```

Comment:
I think it should be changed into "Technical replicate(s)" to be similar to the behavior of biological replicates, while files.replicate.tech_rep was ONLY one, after the change to the calculated property it could be a list